src/pages/Home.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
Please, make a folder for this file named "Home" and rename the file index.js

Reply:
I will implement the requested changes and amend it to this PR.